src/CollectSFDataGui/Server/Startup.cs

```diff
@@ -22,12 +24,16 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             {
                 app.UseDeveloperExceptionPage();
                 app.UseWebAssemblyDebugging();
+
+                app.UseSwagger();
+                app.UseSwaggerUI(c => c.SwaggerEndpoint("/swagger/v1/swagger.json", "WebApplication1 v1"));
```

Comment:
can you give swagger endpoint a real name like CollecsfdataGuiEndpoint or something?

Reply:
suggestion is added